tests/unit/operations_v1/test_operations_rest_client.py

```diff
@@ -82,16 +87,68 @@
 }
 
 
+if GOOGLE_AUTH_AIO_INSTALLED:
```

Comment:
It seems like there's common code in the `if` and the `else`, os it might be clearer to run the common code unconditionally and then `if GOOGLE_AUTH_AIO_INSTALLED`, append the async-specific stuff.

Reply:
I will take a look into this and clean it up. Thanks for pointing that out!